### PR TITLE
[BOARD] Add new variant: TWC_Mesh 

### PR DESCRIPTION
**Pull request by SCWhite**
HI, this version has been made for the local Meshtastic community.
Aiming for an open-source tool for the public, in case of dealing with natural disasters or emergencies.

This variant use the Adafruit Feather nRF52840 Express, combined with the SX1262 Lora module, SSD1306 OLED, and ublox NEO-6M.

### CODE REVIEW

**Review by caveman99 — APPROVED** [review]

**Comment by caveman99** [comment]
please run "trunk fmt" on your branch. Note that on Windows not all linters are available, so you might need to use WSL

**Comment by SCWhite** [comment]
> please run "trunk fmt" on your branch. Note that on Windows not all linters are available, so you might need to use WSL

Thanks!  I was wondering why I can't get pass certain test, you're a life saver

**Comment by markbirss** [comment]
@SCWhite 

you need to change your variant name to more correctly relate to the board used, not nordic\_pca10059 already used by another variant

**Comment by SCWhite** [comment]
> @SCWhite 
> 
> you need to change your variant name to more correctly relate to the board used, not nordic_pca10059 already used by another variant

Thanks or the reminder, I must misunderstood something.  
I'll change it right away, really appreciate!

File: variants/TWC_mesh_v4/variant.h
Comment (markbirss) [inline variants/TWC_mesh_v4/variant.h:1]: the change in platformio.ini for TWC\_mesh\_v4 also must reflect here 

#ifndef \_VARIANT\_NORDIC\_PCA10059\_  
#define \_VARIANT\_NORDIC\_PCA10059\_

e.g

#ifndef _VARIANT\_TWC\_MESH\_v4\__

#define \_VARIANT\__TWC\_MESH\_v4\__

and you still need to submit a protobuf entry for the new hw refer to 

https://meshtastic.org/docs/development/firmware/build/#adding-custom-hardware

Comment (SCWhite) [reply variants/TWC_mesh_v4/variant.h:1]: wow, thanks for the detailed explain.
I just sent out a pull request for protobuf repo.

I'm trying to follow the instruction from [adding-custom-hardware](https://meshtastic.org/docs/development/firmware/build/#adding-custom-hardware) page, but I think I totally misunderstand step 2 & 3
I should send a PR on protobuf first, then add new boards.

Thanks for your patient for me, I'll modify my file accordingly!

**Pull request merged**